### DISCUSSION TITLE
Where to setup memos team members

### DISCUSSION THREAD

**Discussion by yanghugh1983** [usememos/memos#3389] [Q&A] [answered]
I found this description in Getting Started guide:
"Memos can be set to public, workspace, or private. Public memos are visible to everyone. Workspace memos are only visible to you and your team. Private memos are only visible to you."

I want to post workspace memos, but cannot find where to set up team members. Could someone help?

**Comment by boojack** [comment, marked as answer]
Setting -> Member

<img width="1006" alt="image" src="https://github.com/usememos/memos/assets/24653555/1008ecb6-c282-4dba-8938-5472db11f3cd">

**Reply by yanghugh1983** [reply]
Thank you a great deal, boojack!
